docs/src/components/ComponentDoc/ComponentPerfExample/ComponentPerfChart.tsx

```diff
@@ -0,0 +1,40 @@
+import { Box, Flex, Text } from 'src/index'
```

Comment:
```suggestion
import { Box, Flex, Text } from '@stardust-ui/react'
```

Lint rule will be introduced with ESLint